app.py

```
@@ -58,36 +60,88 @@ async def login(user: UserLogin, db: Session = Depends(get_db)):
+    ).all()
+
+    return diaries_today
+
+# 기록 수정
+class UpdateTempDiary(BaseModel):
+    title: str = None
+    content: str = None
+    created_at: datetime = None
+
+@app.put("/api/temp_diary/{temp_diary_id}")
+async def update_temp_diary(temp_diary_id: int, data: UpdateTempDiary, db: Session = Depends(get_db)):
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)